### PR TITLE
chore(override): clean up CDK transformed template

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/override/cdk.go [hunk 1]
```diff
@@ -133,9 +136,40 @@ func (cdk *CDK) transform(body []byte) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func (cdk *CDK) cleanUp(body []byte) ([]byte, error) {
-	// TODO(efekarakus): Implement me.
-	return body, nil
+// cleanUp is removes YAML additions that get injected by the CDK that are unnecessary,
```
Comment (Lou1415926) [inline internal/pkg/override/cdk.go]: nit:
```suggestion
// cleanUp removes YAML additions that get injected by the CDK that are unnecessary,
```

File: internal/pkg/override/cdk.go [hunk 2]
```diff
@@ -133,9 +136,40 @@ func (cdk *CDK) transform(body []byte) ([]byte, error) {
...
+		Parameters               map[string]yaml.Node `yaml:"Parameters,omitempty"`
+		Mappings                 yaml.Node            `yaml:"Mappings,omitempty"`
+		Conditions               yaml.Node            `yaml:"Conditions,omitempty"`
+		Transform                yaml.Node            `yaml:"Transform,omitempty"`
+		Resources                yaml.Node            `yaml:"Resources,omitempty"`
+		Outputs                  yaml.Node            `yaml:"Outputs,omitempty"`
+	}
+	var body template
+	if err := yaml.Unmarshal(in, &body); err != nil {
+		return nil, fmt.Errorf("unmarsal CDK transformed YAML template: %v", err)
```
Comment (Lou1415926) [inline internal/pkg/override/cdk.go]: nit:
```suggestion
		return nil, fmt.Errorf("unmarsal CDK transformed YAML template: %w", err)
```

File: internal/pkg/override/cdk.go [hunk 3]
```diff
@@ -133,9 +136,40 @@ func (cdk *CDK) transform(body []byte) ([]byte, error) {
...
+	body.Description = fmt.Sprintf("%s using AWS Copilot and CDK.", strings.TrimSuffix(body.Description, "."))
+
+	// Get rid of CDK parameters.
+	delete(body.Parameters, "BootstrapVersion")
+
+	out := new(bytes.Buffer)
+	encoder := yaml.NewEncoder(out)
+	encoder.SetIndent(2)
+	if err := encoder.Encode(body); err != nil {
+		return nil, fmt.Errorf("marshal cleaned up CDK transformed template: %v", err)
```
Comment (Lou1415926) [inline internal/pkg/override/cdk.go]: nit:
```suggestion
		return nil, fmt.Errorf("marshal cleaned up CDK transformed template: %w", err)
```

**Review by Lou1415926 — APPROVED** [review]
LGTM!

**Review by dannyrandall — APPROVED** [review]

**Pull request merged**